molecule/driver/dockerdriver.py

```diff
@@ -23,6 +23,7 @@
 import json
 
 import docker
+from docker.utils import kwargs_from_env
```

Comment:
This is already imported as docker.  You're safe to simply use `docker.utils.kwargs_from_env()` below.